src/Nethermind/Nethermind.Consensus/Stateless/NoCacheCodeInfoRepository.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Frozen;
+using System.Data;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Specs;
+using Nethermind.Evm;
+using Nethermind.Evm.CodeAnalysis;
+using Nethermind.Evm.EvmObjectFormat;
+using Nethermind.Evm.State;
+
+namespace Nethermind.Consensus.Stateless;
+
+/// <summary>
+/// Same implementation as CodeInfoRepository but without cache so that it always goes through
+/// the world state, which captures touched bytecodes.
+/// </summary>
```

Comment:
Those are generally `<remarks> ` not `<summary>`, you are using the docs a bit wrong in multiple files.